File: control-assurance-sp/src/types/IAPAssignment.ts
import { IEntity } from "./Entity";

export interface IIAPAssignment extends IEntity {
    IAPActionId?: number;
    UserId?: number;
    GroupNum?: number;
}

export class IAPAssignment implements IIAPAssignment {
    public ID = 0;
    public Title = null;
    public IAPActionId = null;
    public UserId = null;
    public GroupNum = null;
}

File: ControlAssuranceAPI/Models/IAPAssignment.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class IAPAssignment
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public int? IAPActionId { get; set; }

    public int? UserId { get; set; }

    public int? GroupNum { get; set; }

    public DateTime? DateAssigned { get; set; }

    public virtual IAPAction? IAPAction { get; set; }

    public virtual User? User { get; set; }
}
